### PR TITLE
Updated Grotesque Guardians Plugin

### PR DESCRIPTION
**Pull request by f0rmatme**
Added prayer overlay

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java [hunk 1]
```diff
@@ -50,11 +76,96 @@
...
+
+	@Subscribe
+	public void onGameTick(final GameTick event)
+	{
+		final ArrayList<Integer> regions = new ArrayList<Integer>();
+		for (final int intValue : client.getMapRegions())
+		{
+			regions.add(intValue);
+		}
+		if (regions.contains(6727))
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java]: GARGOYLES_REGION is already defined, use it here.

File: runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansTextOverlay.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import net.runelite.api.Perspective;
+import net.runelite.api.coords.LocalPoint;
+import net.runelite.client.game.SpriteManager;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayLayer;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.OverlayPriority;
+import net.runelite.client.ui.overlay.components.TextComponent;
+
+public class GrotesqueGuardiansTextOverlay extends Overlay
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansTextOverlay.java]: Why two overlays?

File: runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java [hunk 3]
```diff
@@ -29,19 +29,45 @@
...
+import javax.annotation.Nullable;
+import net.runelite.api.Client;
+
 
 @PluginDescriptor(
 	name = "Grotesque Guardians",
-	description = "Display tile indicators for the Grotesque Guardian special attacks",
-	tags = {"grotesque", "guardians", "gargoyle", "garg"},
+	description = "Show various helpful utitiles during the Grotesque Gaurdians (Gargoyles) fight",
+	tags = { "bosses", "combat", "minigame", "overlay", "prayer", "pve", "pvm" },
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java]: not sure gargoyles are a minigame :P

File: runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java [hunk 4]
```diff
@@ -50,11 +76,96 @@
...
 	}
+
+	@Nullable
+	DuskAttack getPrayAgainst()
+	{
+		return prayAgainst;
+	}
+
+	@Nullable
+	NPC getDusk()
```
Comment (Ganom) [inline runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java]: Most of our plugins use lombok getters, its best to stay in that format.

Comment (f0rmatme) [reply runelite-client/src/main/java/net/runelite/client/plugins/grotesqueguardians/GrotesqueGuardiansPlugin.java]: not applicable

**Pull request merged**